frontend/src/pages/org/crawls-list.ts

```diff
@@ -272,84 +272,85 @@ export class CrawlsList extends LiteElement {
             <sl-icon name="search" slot="prefix"></sl-icon>
           </sl-input>
         </div>
+          <div class="grow flex items-center mr-2">
```

Comment:
Is the `mr-2` correctly to increase the right margin for the "View" filter? This creates uneven spacing between the grid items when all in one row (large screens) is that intentional?